lib/cli.js

```diff
@@ -25,6 +25,7 @@ const cli = meow({
           -s, --skip-unused     Skip check for unused packages.
           -p, --production      Skip devDependencies.
           -E, --save-exact      Save exact version (x.y.z) instead of caret (^x.y.z) in package.json.
+          -i, --installer <npm> Change npm installer. (eg 'pnpm', 'ied')
```

Comment:
I'd like to reserve `-i` for future use in case users want an option to have `npm-check` automatically install updates.

Do you feel like users who want to use an alternate installer might *always* want to use that other installer? Maybe it should be set as an environment variable so it doesn't need to be added to the command line every time?

Something like

```sh
NPM_CHECK_INSTALLER=pnpm
```

in your `.bashrc`.

What do you think about that?